### DISCUSSION TITLE
modoboa enable pop3

### DISCUSSION THREAD

**Discussion by draconMo** [modoboa/modoboa#3347] [Help]
debian12
postgres
 Modoboa 2.2.0
 
 it was necessary to add pop3 (this protocol was not specified during the modoboa installation process)
Did the following

apt install dovecot-pop3d/bookworm-backports

Have I done everything correctly? Are any other settings required?

**Comment by tonioo** [comment]
It should be enough. Have you checked that it works?

**Reply by draconMo** [reply]
install pop3 , the mail server rebooted. 
When I send a letter, I receive messages in the logs about sending, but the letter does not arrive on gmail. 
 11:32:28 mail postfix/smtp[450367]: A6B47C31E3: to=<dracon753@gmail.com>, relay=gmail-smtp-in.l.google.com[64.233.161.27]:25, delay=1.2, delays=0.1/0.02/0.67/0.42, dsn=2.0.0, status=sent (250 2.0.0 OK  1732696348 2adb3069b0e04-53dd9f9f8f9si3810282e87.73 - gsmtp)
Сould this problem be related to the pop3 installation?

**Reply by tonioo** [reply]
No. Looks like your message is being rejected by Gmail... Have you checked your spam folder?
And have you enabled SPF/DKIM for your domain ?

**Comment by draconMo** [comment]
![image](https://github.com/user-attachments/assets/a8ac26c8-0d7d-416f-9cbd-d5ca67203e7e)

![image](https://github.com/user-attachments/assets/82076ccd-cf8c-4407-af95-e712b90411e4)

It really ends up in spam, I wonder why that is?

**Comment by tonioo** [comment]
Good question... Don't you have more information coming from google? Have you tried their postmaster tools?

**Comment by draconMo** [comment]
postmaster tools
![image](https://github.com/user-attachments/assets/b7ed5f2a-0825-4907-8e20-7c0d566d467f)
spf и dkim
TXT
v=spf1 ip4:ip postfix server a mx ~all

mail._domainkey.andreydracon.ru
v=DKIM1; h=sha256; k=rsa;p=here is the key that was taken from the moodoboa interface

![image](https://github.com/user-attachments/assets/4dda0986-6371-4a7a-8a7b-299ff41e25bd)

I don't understand what I did wrong

**Reply by tonioo** [reply]
Could you translate the first message?

**Reply by draconMo** [reply]
Sorry
SPF and DKIM Authentication
Needs some work - Set up both SPF and DKIM authentication
SPF technology prevents spammers from sending unauthorized emails on behalf of your domain. Recipient servers use DKIM to verify that the email was actually sent by the domain owner.

I don't understand what the problem is, there are records in DNS

**Comment by draconMo** [comment]
figured out the problem - I placed an spf record in the mail.andreydracon.ru subdomain (it should have been in andreydracon.ru)
